src/ros_interop/mission_manager.py

```diff
@@ -0,0 +1,53 @@
+import rospy
+from ros_interop.srv import MissionResponse
+from ros_interop.msg import * 
+from geographic_msgs.msg import GeoPoint
+
+class MissionManager():
```

Comment:
Standardize, either put parentheses (and put `object` between them) or dont put them at all